src/App.tsx

```diff
@@ -1,173 +1,106 @@
-import { Loader } from './components/Loader';
-
-import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">
-              There are no people on the server
-            </p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import React, { useEffect, useState } from 'react';
+import {
+  useNavigate, Route, Routes, NavLink,
+} from 'react-router-dom';
+import cn from 'classnames';
+import { HomePage } from './components/homePage';
+import { People } from './components/peoplePage';
+import { getPeople } from './api';
+import { Person } from './types';
+import { NotFoundPage } from './components/notFoundPage';
+
+export const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const navigate = useNavigate();
+  const [isLoading, setLoading] = useState(true);
+  const [loadingError, setLoadingError] = useState(false);
+
+  async function peopleArr() {
```

Comment:
Use arrow functions to improve readability. Rename the function to show that it's an action `fetchPeopleAsync` for example